pawtectProject/pawtectApp/controller/PetsController.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth.models import User
+from pawtectApp.models import Pet
+from datetime import datetime
+from pawtectApp import utils
+import locale
+class PetsController():
+    def create_pet(self,petInfo,userProfile,myfile):
+        locale.setlocale(locale.LC_ALL, '')
+        birthDate = datetime.strptime(petInfo['birthDate'], '%B %d, %Y')
+        imageUrl = utils.make_image_url(myfile)
+        petObj = Pet()
+        petObj.name = petInfo['name']
+        petObj.picture = imageUrl
+        petObj.microchip_Number = petInfo['microchip_Number']
+        petObj.breed = petInfo['breed']
+        petObj.species = petInfo['species']
+        petObj.birthDate = birthDate
+        petObj.gender = petInfo['gender']
```

Comment:
can we have optional params like this 
```suggestion
        petObj.gender =petObj.gender = petInfo.get('gender',None)
```

or 
```suggestion
       petObj.gender = petInfo.get('gender', '')
```